docs/en/latest/references/apisix_route_v2.md

```diff
@@ -77,11 +77,11 @@ The table below describes each of the attributes in the spec. The fields `apiVer
 | stream[].name                        | string (required)  | Name of the rule.                                                                                                                                                                         |
 | stream[].match                       | object (required)  | Conditions to match the request with the Route.                                                                                                                                           |
 | stream[].match.ingressPort           | integer (required) | Listening port in the Ingress proxy server. This port should be defined in the [APISIX configuration](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L101).        |
-| stream[].backend                     | object             | Backend service (deprecated). Use `http[].backends` instead.                                                                                                                              |
-| stream[].backend.serviceName         | string             | Name of the backend service (depricated). The service and the `ApisixRoute` resource should be created in the same namespace.                                                             |
-| stream[].backend.servicePort         | integer or string  | Port number or the name defined in the service object of the backend (depricated).                                                                                                        |
-| stream[].backend.resolveGranularity  | string             | See [Service resolution granularity](#service-resolution-granularity) for details (depricated).                                                                                           |
-| stream[].backend.subset              | string             | Subset for the target service (depricated). Should be pre-defined in the `ApisixUpstream` resource.                                                                                       |
+| stream[].backend                     | object             | Backend service (depreciated). Use `http[].backends` instead.                                                                                                                              |
```

Comment:
Hi, I think `deprecated` is correct 🤔

Reply:
Yes. I thought I was wrong and changed it... Deprecated is the correct one. I have checked.

I have reverted the commit